### DISCUSSION TITLE
Metagenome: Co-Assembly and Hybrid

### DISCUSSION THREAD

**Discussion by mscarbor** [ablab/spades#1131] [Q&A]
Hello! I am about to embark on assembling 2x150 bp Illumina and Nanopore reads. There are reads form multiple samples that I would like to co-assemble, for both Ilumina and Nanopore. What is the best way to input these reads for this situation? Should each sample be its own "library" or should I just concatenate all the Illumina reads and all the Nanopore reads and input them as three files (forward reads, reverse reads, and then Nanopore reads)? Related-- is there a way to input multiple Nanopore read files? Thanks in advance!

**Comment by asl** [comment]
Hello

You can specify multiple files for a single library. This works as a "concatenation on fly". metaSPAdes supports only single paired-end library
